wicket-kendo-ui-culture/pom.xml

```diff
@@ -25,6 +25,12 @@
 			<artifactId>wicket-jquery-ui-core</artifactId>
 			<version>6.20.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>6.20.1-SNAPSHOT</version>
+			<type>bundle</type>
```

Comment:
Look over the main POM file of this project and notice it is using the maven-bundle-plugin to create the bundle that is in turn used to upload to Maven Central repository (usually through a proxy).

How did you build your local project that you ended up with a bundle instead of the normal jars?

That's your real problem...